apisix/plugins/key-auth.lua

```diff
@@ -32,6 +32,10 @@ local schema = {
             type = "string",
             default = "apikey",
         },
+        query = {
```

Comment:
Should add some constraints to indicate that `query` and `header` are exclusive.